bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -351,29 +343,13 @@ var Tabzilla = (function (Tabzilla) {
     };
     Infobar.prototype.trackEvent = function (action, label, value,
```

Comment:
As it's no longer doing anything, should we just remove this function (and all calls to it)?

Also, the change to this function breaks functionality, as callbacks are never fired. We'll need to make sure the callbacks currently passed to this function get executed.